Multiplexor.cpp

```diff
@@ -0,0 +1,45 @@
+/* ROBORREGOS MAZE 2020.
+ * This multiplexor class is to give a ID number
+ * to each I2C sensor.
+ * To get more information, go to Multiplexor.h file.
+*/
+#include "Multiplexor.h"
+
+Multiplexor::Multiplexor() {}
+
+void Multiplexor::tcaselect(uint8_t number)
+{
+  if (number > 7)
+    return;
+
+  Wire.beginTransmission(TCAADDR);
+  Wire.write(1 << number);
+  Wire.endTransmission();
+  delay(kWaitToSend);
+}
+
+void Multiplexor::setup()
+{
+    while (!Serial);
```

Comment:
What does this mean?

Reply:
It's only to check to check the I2C Multiplexor scanner, I'm not going to use it.

Reply:
I mean the specific line while(!Serial);

Reply:
I mean the specific line while(!Serial);